modules/fscloud/main.tf

```diff
@@ -82,10 +82,12 @@ resource "time_sleep" "wait_for_cis_instance" {
 }
 
 module "cis_domain_settings" {
-  source          = "../../modules/waf"
-  depends_on      = [time_sleep.wait_for_cis_instance]
-  cis_instance_id = module.cis_instance.cis_instance_id
-  domain_id       = module.cis_instance.cis_domain.domain_id
-  enable_waf      = true
+  source                                 = "../../modules/waf"
+  depends_on                             = [time_sleep.wait_for_cis_instance]
+  cis_instance_id                        = module.cis_instance.cis_instance_id
+  domain_id                              = module.cis_instance.cis_domain.domain_id
+  enable_cis_managed_ruleset             = true
+  enable_cis_exposed_creds_check_ruleset = true
+  enable_cis_owasp_core_ruleset          = true
```

Comment:
fscloud recommends to enable `waf` but not all ruesets. Hence we can have variables to enable or disable.